.github/workflows/project-creation.yaml

```diff
@@ -0,0 +1,115 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Project creation
+
+concurrency:
+  group: ${{ github.ref }}-project-creation
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    # Run only on branches/commits and not tags
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-project-creation:
+    name: Check project creation
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        example: ["", "-e seat-adjuster"]
+      fail-fast: false
+
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: latest
+
+      - name: Checkout CLI
+        uses: actions/checkout@v4
+        with:
+          repository: eclipse-velocitas/cli
+          path: cli
+          ref: v0.6.1
+
+      - name: Checkout SDK repo
+        uses: actions/checkout@v4
+        with:
+          path: sdk
+
+      - name: Install CLI
+        shell: bash
+        run: |
+          cd cli
+          npm i && npm run build && npx oclif manifest && npm i -g .
+
+      - name: Create project
+        shell: bash
+        run: |
+          git config --global user.name "Git User"
+          git config --global user.email "gituser@email.com"
+
+          # overwrite SDK path in order to use scripts from this branch (rather than sdk/main)
+          export VELOCITAS_SDK_PATH_OVERRIDE=$(pwd)/sdk
+
+          mkdir app
+          cd app
+
+          # show available commands and version of CLI
+          velocitas
+
+          # copy "full" package-index.json from CLI repo
+          cp ../cli/testbench/test-create/vehicle-app-template/package-index.json ./package-index.json
+
+          velocitas create -n MyApp -l python ${{ matrix.example }}
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          pip install -r ./sdk/.project-creation/test/requirements.txt
+
+      - name: Run verification on generated project
+        shell: bash
+        run: |
+          export VELOCITAS_SDK_ROOT=./sdk
+          export VELOCITAS_APP_ROOT=./app
+
+          # debug print contents
+          ls -al $VELOCITAS_APP_ROOT
+
+          pytest --ignore-glob=**/.devcontainer/* --ignore-glob=**/.skeleton/* ./sdk/.project-creation
+
+      - name: Check if devContainer starts up properly and app is usable
+        uses: devcontainers/ci@v0.3
+        with:
+          subFolder: ./app
+          runCmd: |
+            git init && \
+              pre-commit run --all-files && \
+              pip3 install -r .devcontainer/tests/automated_tests/requirements.txt && \
+              pytest -sx .devcontainer/tests/automated_tests
```

Comment:
```suggestion
            git init && \
            pre-commit run --all-files && \
            pip3 install -r .devcontainer/tests/automated_tests/requirements.txt && \
            pytest -sx .devcontainer/tests/automated_tests
```

Reply:
Since its a continuation of the bash command, just broken to multiple lines, it is common convention to indent the concatenated commands. I would like to keep this pattern here as well.

Reply:
ok fine for me, thought its a mistake